compiler/front/commands.nim

```diff
@@ -80,6 +78,27 @@ type
     passCmd2,                 # second pass over the command line
     passPP                    # preprocessor called processCommand()
 
+# TODO: temporary, move into `msgs` or `commands`
```

Comment:
```suggestion
```
I believe the TODO doesn't apply anymore?

Reply:
I'm not quite sure about the current home. 😕